asciidoc/modules/con_mg_changes-in-vertx-jdbc-client.adoc

```diff
@@ -0,0 +1,8 @@
+[id="changes-in-vertx-jdbc-client_{context}"]
+= Changes in {VertX} JDBC client
+
+The following section describes the changes in {VertX} JDBC client.
+
+== Merged Vert.x SQL common module in JDBC client module
+
+The `io.vertx:vertx-sql-common` module has been merged in `io.vertx:vertx-jdbc-client` module.
```

Comment:
@vietj Do we want to add more details to this note? How will this change effect the users?

Reply:
I think we can add.

_Therefore you should remove any `io.vertx:vertx-sql-common` dependency from your build file since `io.vertx:vertx-jdbc-client` will include it._